master/internal/api_runs_intg_test.go

```diff
@@ -586,12 +620,23 @@ func TestMoveRunsMultiTrialNoSkip(t *testing.T) {
 
 	resp, err = api.SearchRuns(ctx, req)
 	require.NoError(t, err)
-	require.Len(t, resp.Runs, 2)
+	require.Len(t, resp.Runs, 3)
 	// Check if other run moved as well
 	require.Equal(t, runID2, resp.Runs[1].Id)
 	// Check if runs in same experiment
 	require.Equal(t, expID, resp.Runs[0].Experiment.Id)
 	require.Equal(t, expID, resp.Runs[1].Experiment.Id)
+
+	i := strings.Index(resp.Runs[0].LocalId, "-")
+	localID1 := resp.Runs[0].LocalId[i+1:]
```

Comment:
Would it be worth turning this into a utility helper function?

Reply:
The uses are slightly different in other functions and only about 2 lines. I think it's fine for now as only this test is checking multiple Local Ids